tests/xsd_functions/construct_date-02.srx

```diff
@@ -4,9 +4,9 @@
 	<variable name="date"/>
 </head>
 <results>
-		<result></result>
-		<result></result>
-		<result></result>
-		<result></result>
+		<result><binding name="date"><literal datatype="http://www.w3.org/2001/XMLSchema#date">2000-00-01</literal></binding></result>
+		<result><binding name="date"><literal datatype="http://www.w3.org/2001/XMLSchema#date">2000-13-01</literal></binding></result>
+		<result><binding name="date"><literal datatype="http://www.w3.org/2001/XMLSchema#date">2000-06-00</literal></binding></result>
+		<result><binding name="date"><literal datatype="http://www.w3.org/2001/XMLSchema#date">2000-06-32</literal></binding></result>
```

Comment:
The [lexical form for date components](https://www.w3.org/TR/xmlschema11-2/#nt-yrFrag) says:

```
[57]   monthFrag ::= ('0' [1-9]) | ('1' [0-2])
[58]   dayFrag ::= ('0' [1-9]) | ([12] digit) | ('3' [01])
```

I'm not sure how we could expect to get these xsd:date values out of the constructor for these string values:

```
"2000-00-01"
"2000-13-01"
"2000-06-00"
"2000-06-32"
```

Reply:
Agreed, in retrospect, given the syntactic limitations.